packages/@webex/plugin-meetings/src/multistream/receiveSlotManager.ts

```diff
@@ -12,7 +12,7 @@ import {CSI, ReceiveSlot} from './receiveSlot';
  * so this manager has a pool in order to re-use the slots that were released earlier.
  */
 export class ReceiveSlotManager {
-  private allocatedSlots: {[key in MediaType]: ReceiveSlot[]};
+  allocatedSlots: {[key in MediaType]: ReceiveSlot[]};
```

Comment:
I think it would be better to add a public function like findReceiveSlotBySsrc() or something like that instead of making allocatedSlots public. We don't want allocatedSlots to be modified by anything outside ReceiveSlotManager. Also then it's easier to unit test the new public method.

Reply:
Updated.